assets/templates/fragments/field-error-bottom.tmpl

```diff
@@ -0,0 +1,3 @@
+{{/* Use with fragments/field-error-top */}}
+</div>
```

Comment:
Do you think indenting this would bring better clarity?

Reply:
Can't hurt